dask/dataframe/tests/test_rolling.py

```diff
@@ -61,20 +62,33 @@ def rolling_tests(p, d):
     # Test with kwargs
     eq(p.rolling(3, min_periods=2).sum(), d.rolling(3, min_periods=2).sum())
 
+def test_rolling_functions_series():
```

Comment:
You'll probably have to add the following decorator

```python
@pytest.mark.skipif(LooseVersion(pd.__version__) <= '0.18.0',
                    reason="rolling object not supported")
```

See `dask/dataframe/tests/test_dataframe.py` for examples.